rpm_spec/subpackages/manageiq-core

```diff
@@ -32,11 +36,28 @@ done
 
 %post core
 %{__cp} -f %{app_root}/config/cable.yml.sample %{app_root}/config/cable.yml
+# These files are not owned by the rpm.
+#  For upgrades, ensure they have the correct group privs
+#  so root and manageiq users can read them.
+%{__chgrp} manageiq %{app_root}/certs/v2_key %{app_root}/log/*.log
+%{__chgrp} manageiq %{app_root}/tmp/pids/*.pid %{app_root}/config/*.yml
+%{__chmod} g+r,o-rw %{app_root}/certs/v2_key
+%{__chmod} g+r,o-rw %{app_root}/config/*.yml %{app_root}/tmp/pids/*.pid
+%{__chmod} g+rw %{app_root}/log/*.log
 
 %files core
-%defattr(-,root,root,775)
+%defattr(-,root,root,-)
 %{app_root}
 %config(noreplace) %{app_root}/certs
+%attr(-,-,manageiq) %{app_root}/certs
+%attr(-,-,manageiq) %{app_root}/config
+%attr(-,-,manageiq) %{app_root}/log
+%attr(-,-,manageiq) %{app_root}/tmp
+# this is until we fix the code that auto writes this
+# give kbrock a hard time if it is 2022 and this is still in here
+%attr(0664,-,-) %{app_root}/tmp/cache/sti_loader.yml
```

Comment:
this is a separate PR. Want to get these other fixes in before tracking down why the descendant loader wants to modify the `sti_loader`

Had to chmod it `664` so `manageiq` group can rewrite it.

NOTE: yes, the intent was that this file would be part of the rpm and never change, but alas, here we are